tests/parse.spec.js

```diff
@@ -232,4 +232,54 @@ describe('Parse', function() {
     ]);
   });
 
+  it('should handle unknown escape sequences inside a value', function() {
+    m.chai.expect(parse([
+      'device: /dev/disk1',
+      'description: "\\]=-01`234567890=-098e"'
+    ].join('\n'))).to.deep.equal([
+      {
+        device: '/dev/disk1',
+        description: ']=-01`234567890=-098e'
+      }
+    ]);
+  });
+
+  it('should delete known escape sequences', function() {
```

Comment:
I think the changes are complementary. Your changes make the linux script converts things like `\011` into ASCII escape sequences that can be removed here, but do you think we can move that sed logic here, so we make sure it gets applied everywhere?

> (and if TAB is a special-case, maybe this test should be called it('should delete known escape sequences except TAB', function() { ?)

Fixed!